> @ZhaoxiangJin
> 
> drivers.eeprom.build v4.4.0-7451-g5b3901b7d188 Build failure frdm_mcxa577/mcxa577 FAILED:
> 
> ```
> zephyr/kobject_prebuilt_hash.gperf /home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/twister-out/frdm_mcxa577_mcxa577/zephyr_gnu/tests/drivers/build_all/eeprom/drivers.eeprom.build/zephyr/kobject_prebuilt_hash.gperf cd /home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/twister-out/frdm_mcxa577_mcxa577/zephyr_gnu/tests/drivers/build_all/eeprom/drivers.eeprom.build/zephyr && /home/jenkins/agent/workspace/zephyr_test_pipe/.venv/bin/python3 /home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py --kernel /home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/twister-out/frdm_mcxa577_mcxa577/zephyr_gnu/tests/drivers/build_all/eeprom/drivers.eeprom.build/zephyr/zephyr_pre0.elf --gperf-output kobject_prebuilt_hash.gperf --include-subsystem-list /home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/twister-out/frdm_mcxa577_mcxa577/zephyr_gnu/tests/drivers/build_all/eeprom/drivers.eeprom.build/zephyr/misc/generated/struct_tags.jsonTraceback (most recent call last): File "/home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py", line 1102, in <module> main() File "/home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py", line 1065, in main objs = find_kobjects(elf, syms) ^^^^^^^^^^^^^^^^^^^^^^^^ File "/home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py", line 655, in find_kobjects objs = type_obj.get_kobjects(addr) ^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py", line 329, in get_kobjects return type_env[self.child_type].get_kobjects(addr) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/home/jenkins/agent/workspace/zephyr_test_pipe/zephyr/scripts/build/gen_kobject_list.py", line 277, in get_kobjects objs.update(mt.get_kobjects(addr + (i * mt.size))) ^^^^^^^AttributeError: 'ConstType' object has no attribute 'size
> ```
> 
> imx95_evk_cm7
> 
> ```
> -- Looking for device MIMX9596 in /home/ubuntu/nxp/imx95_evk_19/modules/hal/nxp/mcux/mcux-sdk-ng/devices/
> -- Found device folder: /home/ubuntu/nxp/imx95_evk_19/modules/hal/nxp/mcux/mcux-sdk-ng/devices/i.MX/i.MX95/MIMX9596
> CMake Error at /home/ubuntu/nxp/imx95_evk_19/modules/hal/nxp/mcux/mcux-sdk-ng/devices/i.MX/shared.cmake:7 (include):
>   include could not find requested file:
> 
> /home/ubuntu/nxp/imx95_evk_19/modules/hal/nxp/mcux/mcux-sdk-ng/arch/arm/configuration/imx.cmake
> ```
> 
> and all imx mpu cm4/cm33/cm7 fails

imx m-core build issue should be fixed.
seems frdm_mcxa577 `drivers.eeprom.build` also build failed on main branch.

--- a/west.yml
+++ b/west.yml
@@ -213,7 +213,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 02a097059315d7b9a7f7f7d38ee9c97f906867ce
+      revision: b0d24fd6fd036f54a43721f4b0415df43317d7e0
       path: modules/hal/nxp
       groups:
         - hal

